content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: delfin
+description: "delfin deployment on kubernetes guide."
+tags: ["installation guide", "delfin"] 
+---
+
+## Delfin deployment on kubernetes cluster
+Here are the steps to deploy delfin on a kubernetes  cluster 
+
+### Pre-config (Ubuntu 18.04)
```

Comment:
We can specify the needed number of nodes etc? Also system resources expected. If you are depending on the resources needs from k8s and not at all on delfin, you can point to that as well. But better to give